Kritika_Sachdeva/profile/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Kritika Sachdeva</title>
+
+    <link rel="stylesheet" type="text/css" href="mystyles.css">
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" crossorigin="anonymous">
+
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"  crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"  crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+
+<!--HEADER-->
+<section id="header">
+    <div class="container text-center">
+        <div class="user-box">
+            <img src="pp.jpg">
+            <h1>KRITIKA SACHDEVA</h1>
+            <p>Learner. Believer. Complex.</p>
+        </div>
+    </div>
+
+    <div class="scroll-btn">
+        <div class="scroll-bar"><a href=""> <span></span></a></div>
```

Comment:
what's the purpose of empty link here?

Reply:
It was supposed to be linked to other section on the same page.
Modified the code accordingly.